src/cmds/pbsnodes.c

```diff
@@ -695,6 +697,23 @@ prt_node(struct batch_status *bstat)
 				printf("     %s", pattr->name);
 				if (pattr->resource)
 					printf(".%s", pattr->resource);
+#ifndef WIN32
+				if(strcmp(pattr->name, ATTR_NODE_last_used_time) == 0) {
+					memset(&tm, 0, sizeof(struct tm));
```

Comment:
It would be better if we can test whether this attribute has been set a value using the flag ATR_VFLAG_SET.